### DISCUSSION TITLE
overlap codes

### DISCUSSION THREAD

**Discussion by che-ri** [dawnlabs/carbon#1550] [Q&A] [answered]
**Describe the bug**

**To Reproduce**

1. Go to '…'
2. Click on '…'
3. See error

**Expected behavior**

**Screenshots**

**Info (please complete the following information):**

- OS [e.g. macOS, Linux, Windows, iOS]: macOS
- Browser [e.g. Chrome, Safari, Firefox]: Chrome
- Carbon URL [e.g. carbon.now.sh?bg=pink]: [carbon link](https://carbon.now.sh/?bg=rgba%280%2C0%2C0%2C1%29&t=seti&wt=none&l=dart&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%252F%252F%252Fstatic%250Aclass%2520VendingMachine%257B%250A%2520%2520static%2520getCoke%28%29%2520%257B%250A%2520%2520%2520%2520return%2520%27coke%27%253B%250A%2520%2520%257D%250A%2520%2520static%2520getWater%28%29%257B%250A%2520%2520%2520%2520return%2520%27water%27%253B%250A%2520%2520%257D%250A%257D%250A%250Amain%28%29%257B%250A%2520%2520var%2520coke%2520%253D%2520VendingMachine.getCoke%28%29%253B%250A%2520%2520var%2520water%2520%253D%2520VendingMachine.getWater%28%29%253B%250A%2520%2520print%28coke%29%253B%2520%2520%252F%252Fcoke%250A%2520%2520print%28water%29%253B%2520%252F%252Fwater%250A%257D)
![carbon (48)](https://user-images.githubusercontent.com/76051880/218634872-5fd7cddd-1fc2-4641-8e0e-4ba7271d91bd.png)

<details>
  <summary>Code snippet</summary>
  <pre>
  

```dart
///static
class VendingMachine{
  static getCoke() {
    return 'coke';
  }
  static getWater(){
    return 'water';
  }
}

main(){
  var coke = VendingMachine.getCoke();
  var water = VendingMachine.getWater();
  print(coke);  //coke
  print(water); //water
}
```
  </pre>
</details>

**Comment by angiejones** [comment]
I think this may be related. If not, I'm happy to open another issue.

Lines are showing ok in [Carbon](https://carbon.now.sh/?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=a11y-dark&wt=none&l=text%2Fx-java&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=true&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=if%28salary%2520%253E%253D%2520requiredSalary%29%2520%257B%250A%250A%2520%2520%2520%2520if%28years%2520%253E%253D%2520requiredYearsEmployed%29%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520System.out.println%28%2522You%2520qualify%2520for%2520the%2520loan.%2522%29%253B%250A%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520else%257B%250A%2520%2520%2520%2520%2520%2520%2520System.out.println%28%2522Sorry%252C%2520you%2520do%2520not%2520qualify.%2522%29%253B%250A%2520%2520%2520%2520%257D%250A%257D) but when I export as a png image, they are wrapping and the text is becoming out of order

### Code

```java
if(salary >= requiredSalary) {

    if(years >= requiredYearsEmployed){
        System.out.println("You qualify for the loan.");
    }
    else{
       System.out.println("Sorry, you do not qualify.");
    }
}
```

### Carbon View
![carbon_view](https://user-images.githubusercontent.com/15972783/220001330-50f019ab-a18b-4f25-ba9a-49b6012eb2ed.png)


### Exported View

![carbon](https://user-images.githubusercontent.com/15972783/220001097-c0877e88-9d11-4b3f-8981-af52cd04d594.png)
Notice on line 5, the `}`, `"`, and `);` are all out of order now. Also on line 8, the text is on top of the `}`.

**Comment by angiejones** [comment, marked as answer]
For anyone looking for a workaround:

1. Go to Settings (click gear icon in the top header)
2. Choose **Window**
3. Deselect **auto adjust width**

![workaround](https://user-images.githubusercontent.com/15972783/220002954-9db1cecb-436c-4e9d-a162-93ade9dac37b.png)
